core/src/core.cpp

```diff
@@ -450,8 +450,12 @@ int sdrpp_main(int argc, char* argv[]) {
     bandplan::loadColorTable(bandColors);
 
     bool _maximized = maximized;
-    int fsWidth, fsHeight, fsPosX, fsPosY;
-    int _winWidth, _winHeight;
+    int fsWidth = 0;
```

Comment:
please revert, these values are never used if not assigned (try to understand the rest of the code) and declaring them like this just waste space and is less clean